tests/test_dense.py

```diff
@@ -34,12 +34,12 @@ def test_check_header_valid_values(self):
 
     def test_filter_expression_scores(self):
         scores = ["BRCA1", 4, 0, 3, "0", None, "", "   ", "nan", "NaN", "Nan", "NAN"]
-        cells = ["foo", "foo2", "foo3", "foo4", "foo5", "foo6", "foo7"]
+        cells = ["foo", "foo2", "foo3", "foo4", "foo5"]
         actual_filtered_values, actual_filtered_cells = DenseIngestor.filter_expression_scores(
             scores[1:], cells
         )
         self.assertEqual([4, 3], actual_filtered_values)
-        self.assertEqual(["foo2", "foo4"], actual_filtered_cells)
+        self.assertEqual(["foo", "foo3"], actual_filtered_cells)
```

Comment:
The test here should have another valid value after all the values that are expected to be filtered to demonstrate that filtered values and cells names do not get "out of sync" during filtering because of the non-numeric NaN values.